src/dcp/dcp.c

```diff
@@ -90,7 +91,7 @@ void print_usage(void)
     printf("  -i, --input <file>       - read source list from file\n");
     printf("  -L, --dereference        - copy original files instead of links\n");
     printf("  -P, --no-dereference     - don't follow links in source\n");
-    printf("  -p, --preserve           - preserve permissions, ownership, timestamps, extended attributes\n");
+    printf("  -p, --preserve           - preserve permissions, ownership, timestamps (see also --copy-xattrs)\n");
```

Comment:
Do you mean this instead?
```suggestion
    printf("  -p, --preserve           - preserve permissions, ownership, timestamps (see also --xattrs)\n");
```

Reply:
Yep